.buildkite/commands/publish_to_s3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -eu
+
+# The A8C_LIBS_* credentials are for the `a8c-libs` S3 bucket in a separate `a8c` account.
+# We use distinct names to prevent conflicts with the AWS credentials of our `a8c-apps` account (used for all our other AppsInfra buckets).
```

Comment:
Re-reading this in context, here's another suggestion you might find even clearer for that comment here?

```suggestion
# The A8C_LIBS_* credentials are for the `a8c-libs` S3 bucket in which we want to publish the SDK artifact, and which lives in a separate `a8c` AWS account.
# We don't use the standard AWS env names here to prevent conflicts with the AWS credentials of our main `a8c-apps` account (used for all our other AppsInfra S3 buckets).
```

wdyt?

Reply:
Yes, it's very clear now 👍 thanks!